pubspec.lock

```diff
@@ -42,28 +42,28 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.4.2"
+    version: "2.5.0-nullsafety.1"
```

Comment:
All the packages used are not yet migrated to `null safety`. So I don't think there is a need to migrate to `null saftey ` as of now.

Reply:
`pubspec.lock` changes should never be pushed as it is automatically generated by flutter when building.

Reply:
Should i restore it , and then commit again?